destinations/sift/handler.js

```diff
@@ -0,0 +1,114 @@
+// Segment & Sift 
+const SEGMENT_WRITE_KEY = ""
+const endpoint = "https://api.sift.com/v205/events"
+const endpointScore = endpoint + "?return_score=true"
+const endpointWorkflow = endpoint + "?return_workflow_status=true"
+async function addScores(fields, res) {
+    console.log("scores", res);
+	if (res.status == 0) {
+		var scoreBody = {
+			"userId": fields.$user_id,
+			"traits": {
+				"contentAbuseScore": res.score_response.scores.content_abuse.score,
+				"paymentAbuseScore": res.score_response.scores.payment_abuse.score
+			}
+		}
+		const request = await fetch("https://api.segment.io/v1/identify", {
+			body: JSON.stringify(scoreBody),
+			headers: new Headers({
+				"Authorization": "Basic " + btoa(SEGMENT_WRITE_KEY + ":"),
+				"Content-Type": "application/json"
+			}),
+			method: "post"
+		})
+		return request.json()
+	} 
+}
+async function addDecisions(fields, res) {
+  console.log("decisions", res);
+	var decisionBody = {
+		"userId": fields.$user_id,
+		"traits": {
+			"contentAbuseDecisions": res.score_response.workflow_statuses[0].history[0].config.decision_id,
+			"paymentAbuseDecisions": res.score_response.workflow_statuses[0].history[0].config.decision_id
+		}
+	}
+	const request = await fetch("https://api.segment.io/v1/identify", {
+		body: JSON.stringify(decisionBody),
+		headers: new Headers({
+			"Authorization": "Basic " + btoa(SEGMENT_WRITE_KEY + ":"),
+			"Content-Type": "application/json"
+		}),
+		method: "post"
+	})
+	return request.json()
+}
+async function siftEventCall(fields) {
+	const res = await fetch(endpoint, {
+		body: JSON.stringify(fields),
+		headers: { "Content-Type": "application/json" },
+		method: "post"
+	})
+	const siftResponse = await res.json();
+	if (siftResponse.status <= 0) {
+		// Please implement conditions for retries. 
+	} else if (siftResponse.status >= 0) {
+		throw new InvalidEventPayload(siftResponse.error_message);
+	}
+  var response;
+	if (endpoint == endpointScore) {
```

Comment:
Hey @ssaouma ! This and the following else statement will never execute. What were you trying to do here?

Reply:
Ah, you're right! Meant to left it up to the customers to customize endpoints they want to hit (score/workflow calls vs. just a regular data point). I submitted another pull request to address this issue.